### ISSUE TITLE
"404 Not Found" for data and model required for demo

### ISSUE THREAD

**Issue by Next-RicardoGuerrero** [sebastian-lapuschkin/lrp_toolbox#17]
The url's specified in install.sh are not available anymore.

**Comment by sebastian-lapuschkin** [comment]
hi @Next-RicardoGuerrero , i just checked all the model URLs and found all files to be available for download.

for the python code:
'http://heatmapping.org/files/lrp_toolbox/models/models_mnist_nn.zip' works

for the matlab variant:
'http://heatmapping.org/files/lrp_toolbox/models/models_mnist_mat.zip' works

those models are in fact part of the toolbox if you clone the whole package from github, and to be found in `lrp_toolbox/models`

The pretrained Caffe reference model downloaded in `lrp_toolbox/caffe-master-lrp/demonstrator/download_model.sh`, ie:
http://heatmapping.org/files/lrp_toolbox/models/bvlc_caffe_reference_model/bvlc_reference_caffenet.caffemodel
and
http://heatmapping.org/files/lrp_toolbox/models/bvlc_caffe_reference_model/caffe_ilsvrc12.tar.gz

also exist and can be downloaded.

which are the files you specifically have problems with?

**Comment by Next-RicardoGuerrero** [comment]
Hello Sebastian Lapuschkin,

those urls you provide in the comment works perfect, but the urls in _lrp_toolbox_python/install.sh_ are different.

```
# download and extract the MNIST hand written data set
if ! [[ -f data/MNIST/test_images.npy && -f data/MNIST/test_labels.npy ]]
then
  fname=data_mnist_npy.tar.gz
  wget -nc http://heatmapping.org/files/lrp_toolbox/data/$fname
  tar xvf $fname
  rm $fname
fi

# download and extract the model required for successfully run the demo
if ! [ -f models/MNIST/long-rect.nn ]
then
  fname=models_mnist_nn.tar.gz
  wget -nc http://heatmapping.org/files/lrp_toolbox/models/$fname
  tar xvf $fname
  rm $fname
fi
```

**Comment by Next-RicardoGuerrero** [comment]
Also, I have downloaded http://heatmapping.org/files/lrp_toolbox/models/models_mnist_mat.zip and seems the file `models/MNIST/short-rect.mat` is corrupt, as it's the only one in this zip that can be uncompressed.

**Comment by sebastian-lapuschkin** [comment]
hi @Next-RicardoGuerrero 

i assume you have just copied the lines, ie
`http://heatmapping.org/files/lrp_toolbox/data/$fname`
into your browser or used it as input for `wget`.

Note that those urls do contain the placeholder `$fname` which is to be replaced with a value defined in the immediately preceding line in the shell script.

The urls I postet are the result of a resolution of those placeholders, as it should happen if you just run the shell script as is.

finally, if you just did a `git clone https://github.com/sebastian-lapuschkin/lrp_toolbox` you do not need to download the models again, as they shold already be there in `lrp_toolbox/models/MNIST/`

does that help?

**Comment by sebastian-lapuschkin** [comment]
> Also, I have downloaded http://heatmapping.org/files/lrp_toolbox/models/models_mnist_mat.zip and seems the file `models/MNIST/short-rect.mat` is corrupt, as it's the only one in this zip that can be uncompressed.

I can verify that there is an issue with the decompression of this archive. however, all files *except* `short-rect.mat` should be fine.

**Comment by Next-RicardoGuerrero** [comment]
Not exactly, is not that I just blindly copied the url. `$fname` contains a name of a file compressed in tar.gz but the files are compressed in zip, that's why it doesn't work.

I have just reviewed the repository and seems that this issue I commented seems to be solved. I guess that I was using an outdated version (I got it from [here](http://heatmapping.org/files/lrp_toolbox/code/lrp_toolbox_python.zip)). I will download the repo and use the latest version.

Thank you very much for your help.
